src/plugins/validation/validator.js

```diff
@@ -0,0 +1,54 @@
+import PluginComponent from '../plugin.component';
+import LIVR from 'livr';
+import * as validationService from './validation.service';
+
+LIVR.Validator.defaultAutoTrim(true);
+
+const Plugin = PluginComponent('validator');
+
+class Validator extends Plugin {
+	constructor() {
+		super(...arguments);
+	}
+
+	onInit() {
+		const validationSettings = this.validationSettings;
+		if (validationSettings.canValidate) {
+			this.validator = new LIVR.Validator(validationSettings.rules);
+		}
+	}
+
+	get error() {
+		if (this.validator) {
+			const validated = {
+				[this.key]: this.value
+			};
+			const validData = this.validator.validate(validated);
```

Comment:
can we use only .getErrors()? there is no sence to call validate 2times?

Reply:
Doesn't work like this